lxd-migrate/main.go

```diff
@@ -313,9 +313,9 @@ func askBool(question string, default_ string) bool {
 		if input == "" {
 			input = default_
 		}
-		if shared.StringInSlice(strings.ToLower(input), []string{"yes", "y"}) {
+		if shared.ValueInSlice(strings.ToLower(input), []string{"yes", "y"}) {
```

Comment:
does this build using 5.0's go version?

Reply:
Since we are building 5.0 with the latest Go version, I assumed it was fair game to rely on modern Go version for bits shipped by the snap.